### PR TITLE
document how to run benchmarks with a benches/README.md

### PR DESCRIPTION
**Pull request by martinlindhe**

### CODE REVIEW

**Comment by HeroicKatora** [comment]
Can we move this `Makefile` and possibly also a `Readme.md` to the same effect into the `benches` folder? A top-level `Makefile` could be confusing people into running it instead of `cargo` to build the crate itself.

**Comment by martinlindhe** [comment]
I dropped the makefile in favor for a benches/README.md

File: benches/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Getting started with benchmarking
+
+To run the benchmarks you need a nightly rust toolchain.
+Then you launch it with
+
+    rustup run nightly cargo bench --features=benchmarks
```
Comment (fintelia) [inline benches/README.md]: Couldn't you just do `cargo +nightly bench --features/benchmarks`?

Comment (martinlindhe) [reply benches/README.md]: Wasn't aware of this syntax. I'll update PR

Comment (martinlindhe) [reply benches/README.md]: Sorted

**Pull request merged**